state_test.go

```diff
@@ -318,6 +318,25 @@ func TestStateUpdateReportStatus(t *testing.T) {
 	assert.WithinDuration(t, now2, time.Now(), 3*time.Second)
 
 	maxReportSendingTime = old
+
+	// pretend update was aborted at the backend, but was applied
+	// successfully on the device
+	usr = NewUpdateStatusReportState(update, client.StatusSuccess)
+	sc = &stateTestController{
+		reportError: NewFatalError(client.ErrDeploymentAborted),
+	}
+	s, c = usr.Handle(&ctx, sc)
+	assert.IsType(t, s, &ReportErrorState{})
+	assert.Equal(t, client.StatusSuccess, s.(*ReportErrorState).status)
```

Comment:
Ok, makes sense. But can I request that `status` is renamed to `updateStatus`? That would make it much clearer. Unless it's a huge hassle?